virttest/virt_admin.py

```diff
@@ -702,6 +704,32 @@ def command(cmd, **dargs):
     return ret
 
 
+def check_server_name(server_name="virtproxyd"):
+    """
+    Determine the server name under different daemon mode.
+
+    :param server_name: name of the managed server
+    :return: name of the managed server
+    """
+    if not utils_split_daemons.is_modular_daemon():
+        server_name = "libvirtd"
+    return server_name
+
+
+def managed_daemon_config(conf_type="virtproxyd"):
```

Comment:
Could this be move into utils_config modular for more broader visibility ?

Reply:
@chunfuwen The function just only useful for virt admin cases. 
# virt-admin server-list 
 Id   Name
------------------
 0    virtproxyd
 1    admin

As far as I know, the item of server name is not used in other features. It probably not very good to move this into utils_config